functions/nvm.fish

```diff
@@ -196,7 +207,7 @@ function _nvm_use
         end
     end
 
-    if set -l root (_nvm_find_up (pwd) $nvm_file)
+    if set -l root (_nvm_find_up (pwd) $nvm_file); and test $ver != (_nvm_resolve_version (cat $root/$nvm_file))
```

Comment:
Nice one-liner! But I'd prefer to avoid using `cat`. Can we change it to this?

```fish
if set -l root (_nvm_find_up (pwd) $nvm_file)
    read -l line <$root/$nvm_file
    if test $ver != (_nvm_resolve_version $line)
        echo $argv[1] >$root/$nvm_file
    end
end
```

Reply:
Ah, great.  I didn't like using `cat` either, so thanks for the improvement!